homework.md

```diff
@@ -4,77 +4,80 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
Hi Hozan,
You have a typo here on question 3, 
2 * 3 * 6 ... not 2 _ 3 _ 6 ...

Reply:
sure things, thanks for letting me knwo.